Scripts/TileMap.gd

```diff
@@ -1,43 +1,84 @@
 extends TileMap
 
-signal proportions(tile_scale, grid_size,tile_size)
+signal proportions(grid_data)
+
+
 #this is the size of each tile
-@export var our_tile_size = Vector2i(16,16)
+@export var our_tile_size = Vector2(16,16)
 
 #how many tiles the grid should be
-@export var our_grid_size = Vector2i(12,12)
+@export var our_grid_size = Vector2(12,12)
```

Comment:
Why was this changed?

Reply:
because it makes my git history look more active